help.lisp

```diff
@@ -54,11 +54,44 @@
                                   (:right (format nil "~a~a~a" (if (= c 0) "" padstr) len s)))))))
     (apply 'mapcar 'concat (or cols '(nil)))))
 
+(defun sort-flattened-bindings (keymaps sort-pred)
+  "Returns all bindings in KEYMAPS in a flattened list, globally sorted with SORT-PRED."
+  (let ((all-bindings (flatten (mapcar #'kmap-bindings keymaps))))
+    (sort all-bindings sort-pred)))
```

Comment:
`sort` is a destructive operation, lets copy the list to before sorting to avoid mutating the arguments. 

```suggestion
    (sort (copy-list all-bindings) sort-pred)))
```